watchtower/wtclient/candidate_iterator_test.go

```diff
@@ -100,6 +100,12 @@ func TestTowerCandidateIterator(t *testing.T) {
 	}
 	towerIterator := newTowerListIterator(towerCopies...)
 
+	// The iterator has towers and should report as non-empty.
+	empty := towerIterator.IsEmpty()
+	if empty {
+		t.Fatal("iterator with towers incorrectly reports as empty")
+	}
+
```

Comment:
can you pls use the `require` package rather? if you rebase onto master you will see that the rest of this file has been updated to use `require` rather

Reply:
same below

Reply:
I had started a little self-review and made this same note to myself! Will do.